osu.Game/BackgroundDataStoreProcessor.cs

```diff
@@ -221,6 +225,7 @@ private void processScoresWithMissingStatistics()
                 catch (Exception e)
                 {
                     Logger.Log(@$"Failed to populate maximum statistics for {id}: {e}");
+                    realmAccess.Write(r => r.Find<ScoreInfo>(id)!.TotalScoreUpgradeFailed = true);
```

Comment:
It's a bit weird that the missing statistics population task is also using `TotalScoreUpgradeFailed`, overloading the flag. Like if the flag is specifically about total score upgrades but another reprocessing task is also using it...

Propose renaming the flag to `BackgroundReprocessingFailed` or something along those lines for what is probably the most allowable wide-ranging use of it.

Reply:
Sounds good, I've renamed it.